site/src/components/progress-bar.md

```diff
@@ -0,0 +1,136 @@
+		role="progressbar"
+		class="progress-bar"
+		style="width: 45%;"
+		aria-label="demo progress bar"
+		aria-valuenow="45"
+	></div>
+</div>
+```
+
+Set the size and the shape of the progress bar on the container. Nest the progress bar within the `progress-bar-container` and optionally set the color and indeterminate property on the `progress-bar`.
+
+See below for more examples.
+
+## Size
+
+There are two sizes of the progress bar: `progress-bar-md` / `progress-bar-lg`. The default size is `progress-bar-md`.
+
+```html
+<label class="label margin-top-xs margin-bottom-xs" for="progress-bar-md">Medium</label>
+<div id="progress-bar-md" class="progress-bar-container">
+	<div
+		role="progressbar"
+		class="progress-bar"
+		style="width: 75%;"
+		aria-label="demo size medium progress bar"
+		aria-valuenow="75"
+	></div>
+</div>
+<label class="label margin-top-xs margin-bottom-xs" for="progress-bar-lg">Large</label>
+<div id="progress-bar-lg" class="progress-bar-container progress-bar-lg">
+	<div
+		role="progressbar"
+		class="progress-bar"
+		style="width: 75%;"
+		aria-label="demo size large progress bar"
+		aria-valuenow="75"
+	></div>
+</div>
+```
+
+## Shape
+
+Progress bars can have rounded corners or sharp corners. The default shape is round, but if you want square corners, apply the `progress-bar-square` class.
+
+```html
+<label class="label margin-top-xs margin-bottom-xs" for="progress-bar-rounded">Rounded</label>
+<div id="progress-bar-rounded" class="progress-bar-container progress-bar-lg">
+	<div
+		role="progressbar"
+		class="progress-bar"
+		aria-label="demo round shaped progress bar"
+		aria-valuenow="100"
+	></div>
+</div>
+<label class="label margin-top-xs margin-bottom-xs" for="progress-bar-square">Square</label>
+<div id="progress-bar-square" class="progress-bar-container progress-bar-lg progress-bar-square">
+	<div
+		role="progressbar"
+		class="progress-bar"
+		aria-label="demo square shaped progress bar"
+		aria-valuenow="100"
+	></div>
+</div>
+```
+
+## Indeterminate
+
+Use an indeterminate progress bar to show that an operation is being executed. An indeterminate bar will continue to cycle on repeat, unlike a determinate progress bar.
+
+```html
+<label class="label margin-top-xs margin-bottom-xs" for="progress-bar-indeterminate"
+	>Indeterminate</label
+>
+<div
+	id="progress-bar-indeterminate"
+	class="progress-bar-container progress-bar-lg progress-bar-rounded"
+>
+	<div
+		role="progressbar"
+		class="progress-bar indeterminate"
+		aria-label="demo indeterminate progress bar"
+	></div>
+</div>
+```
+
+## Color
+
+`progress-bar-danger`, `progress-bar-warning`, and `progress-bar-success` will change the color of the bar to indicate a validation state.
+
+```html
+<label class="label margin-top-xs margin-bottom-xs" for="progress-bar-danger">Danger</label>
+<div id="progress-bar-danger" class="progress-bar-container progress-bar-lg">
+	<div
+		role="progressbar"
+		class="progress-bar progress-bar-danger indeterminate"
```

Comment:
I see that the Fluent doc uses indeterminate progress bars in their validation state example. It seems misleading though because if there's an error or success, it would become a static progress bar. I think changing these examples to static bars makes more sense.

Reply:
You can remove the indeterminate class in the logic of the code and the bar would become static. That change doesn't need to happen in atlas, it was just as an example